osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/MultiSpectatorScreen.cs

```diff
@@ -169,11 +175,24 @@ protected override void Update()
                                               .OrderBy(i => Math.Abs(i.SpectatorPlayerClock.CurrentTime - syncManager.CurrentMasterTime))
                                               .FirstOrDefault();
 
+                // Only bind adjustments if there's actually a valid source, else just use the previous ones to ensure no sudden changes to audio.
+                if (currentAudioSource != null)
+                    bindAudioAdjustments(currentAudioSource);
```

Comment:
Out of curiosity, could there be a case where audio adjustments differ between spectated players? Would imagine only having to bind once rather than switching every time the candidate player area changes.

Reply:
Maybe. The reason I did it this way was in the case the original player quit or lagged out.

The main case it's important is time ramp, where the adjustments are not constant.